### PR TITLE
Tudu 54 frontend components

### PR DESCRIPTION
**Pull request by Proliecan**

### CODE REVIEW

**Review by DE38 — APPROVED** [review]
Looks very fancy!

File: src/routes/style.css [hunk 1]
```diff
@@ -29,8 +30,19 @@ a:hover {
...
+	font-size: 70px;
+	line-height: 80px;
+	display: flex;
+	align-items: center;
+
+	/* White */
+	color: #EEEEEE;
+
+	/* Logo Shadow */
+	text-shadow: 0px 1.37056px 0px #B1B1B1;
```
Comment (Proliecan) [inline src/routes/style.css:45]: This gets rendered with the wrong font weight in firefox on windows and I have no clue why. I atomized the issue in `static/Logo-Test.html`, which gets rendered the right way.
There seems to be some interference or some svete-scope stuff I don't understand.
Input needed.
image
image

**Review by Proliecan — COMMENTED** [review]
This cannot be committed as is. the `<h1/>` issue remains.

File: src/routes/style.css (hunk 1 above)
Comment (DE38) [reply src/routes/style.css:45]: I have the same, i don't know why. I would approve it nevertheless.

Comment (Proliecan) [reply src/routes/style.css:45]: I am not satisfied with this thinner font. Have you tested other browsers?

Comment (Proliecan) [reply src/routes/style.css:45]: I just tried Chromium/edge, same results

Comment (DE38) [reply src/routes/style.css:45]: For me it works in chrome (left Firefox/ right Chrome): 
grafik

Comment (Proliecan) [reply src/routes/style.css:45]: Please compare with `/Logo-Test.html`.

**Comment by Proliecan** [comment]
# Todo:
- [ ] Date input
- [ ] Dropdown input
- [ ] Different button colors
- [ ] Text area
- [ ] Heading (css)
- [ ]  label icon combo: image
- [ ] Icon buttons

File: src/routes/style.css (hunk 1 above)
Comment (Proliecan) [reply src/routes/style.css:63]: I seem to have solved the issue in https://github.com/TuDU-SE/TuDu-Frontend/pull/9/commits/eae12559669f5f0c3a1d29a38ea38c92ef605bfc.

**Comment by Proliecan** [comment]
All basic components we should need to build the interface.

Date input is not really satisfactory, but no time to rebuild it from scratch instead of using html components.

**Comment by DE38** [comment]
Sieht richtig gut aus! 👍

**Review by DE38 — APPROVED** [review]

**Pull request merged**